src/set_theory/ordinal/fixed_point.lean

```diff
@@ -1,5 +1,5 @@
 /-
-Copyright (c) 2022 Violeta Hernández Palacios. All rights reserved.
+Copyright (c) 2022 Violeta Hernández Palacios, Mario Carneiro. All rights reserved.
```

Comment:
The date too

Reply:
What year should I add?

Reply:
The relevant commit is 182c3035667a5ccdbd5059738f978e023b006b9c in 2018.

Reply:
Like this?